src/V3Randomize.cpp

```diff
@@ -41,6 +41,61 @@
 
 VL_DEFINE_DEBUG_FUNCTIONS;
 
+// ######################################################################
+// Establishes the target of a rand_mode() call
+
+struct RandModeTarget final {
+    // MEMBERS
+    AstVar* receiverp;  // Variable that is the target of the rand_mode() call
+    AstNodeExpr* fromp;  // Expression from which the rand_mode() call originates
+    AstClass* classp;  // Class in which rand_mode should be set
```

Comment:
```suggestion
    AstVar* const receiverp;  // Variable that is the target of the rand_mode() call
    AstNodeExpr* const fromp;  // Expression from which the rand_mode() call originates
    AstClass* const classp;  // Class in which rand_mode should be set
```

I think if I read this right.  Similar in other struct below.  BTW I'd probably have just made this a class and added the accessors, but I can live with this way too.